Support for different board sizes + coin drop animation

amanpalariya opened the pull request:
I'm confused

Satvinder31415 committed:
Commit #1

Removed redundant double call to diagonal_win() in winning_move()

update logo

update logo

update logo

update logo

Resolved undo issue#13

Generalised some expressions

Generalised some expressions

Works for different board sizes

Added animation and changable board patterns

Update README.md

Create CODE_OF_CONDUCT.md

resolved col bug

Resolved issue#21

run black on the project

Minor bug fixes

Added animation and changable board pattern

Board pattern

Changed a comment

amanpalariya committed:
Merge branch 'master' into master

code-monk08 commented:
@amanpalariya don't be confused, you have done exactly what I asked you to do.

code-monk08 commented:
we have some conflicts here.. please remove them.. also I am very happy with your coin drop animation.. can you keep only keep these changes 

- coin drop animation
- generalizations
- ability to change aesthetics

I am a bit reluctant to merge other changes such as:

1. reduced font size.
2. increased section height.

![Background](https://user-images.githubusercontent.com/19390263/70959832-3fc73780-20a3-11ea-90bc-56590bbbbf96.png)

amanpalariya committed:
Merge branch 'master' into master

amanpalariya committed:
Restored the section size

The message on the top of the screen will always appear in the center for all board sizes

amanpalariya commented:
Done 👍🏻

amanpalariya commented:
If this PR is merged, people working on issue #7 will have to change the code a little because the board is drawn with coloured rectangle + black circle in their branch and I have done it using transparent tileable pattern `board1_100px.png`. Coin drop animation will not work without a transparent board.

code-monk08 commented:
thanks for the amazing job done @amanpalariya 
I will see what I can do while merging in your PR as we have 2 other open PRs for issue 7
